internal/cli/atlas/clusters/create.go

```diff
@@ -84,6 +84,16 @@ func (opts *CreateOpts) newCluster() (*atlas.Cluster, error) {
 		if err := file.Load(opts.fs, opts.filename, cluster); err != nil {
 			return nil, err
 		}
+		// There can only be one
+		if cluster.ReplicationSpecs != nil {
+			cluster.ReplicationSpec = nil
+		}
+		// This can't be sent
+		cluster.MongoURI = ""
+		cluster.MongoURIWithOptions = ""
+		cluster.MongoURIUpdated = ""
+		cluster.StateName = ""
+		cluster.MongoDBVersion = ""
```

Comment:
[q] why are we setting these as empty? Can't we just not set them?

Reply:
when you do describe they are set, if you try to reuse what was output by describe you'll get errors of them being set